core/include/usFrameworkFactory.h

```diff
@@ -48,29 +49,20 @@ class US_Core_EXPORT FrameworkFactory
     FrameworkFactory(void);
     virtual ~FrameworkFactory(void);
 
+    FrameworkFactory(const FrameworkFactory&) = delete;
+    FrameworkFactory& operator=(const FrameworkFactory&) = delete;
+
```

Comment:
I think we should make this class much simpler and remove all the constructors and assignment operators. This class has no state and it would be perfectly okay to copy it around.